src/duiker.py

```diff
@@ -36,7 +36,7 @@ def xdg_data_home(name: Optional[str] = None) -> pathlib.Path:
     Return the XDG Base Directory Specification data directory for a specific
     application, or the base directory itself.
     """
-    home = pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or '~/.local/share'))
+    home = pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or os.path.expanduser('~') + '/.local/share'))
```

Comment:
I'm not clear on what the difference is here. The original call, if `XDG_DATA_HOME` doesn't exist, would be:

```python
home = pathlib.Path(os.path.expanduser('~/.local/share'))
```

Can you paste the traceback of the exception you got?

N.B. the version on PyPI doesn't use this function yet. Are you running Duiker from `HEAD`?